source/Calamari/Kubernetes/Scripts/KubectlBashContext.sh

```diff
@@ -70,10 +72,21 @@ function setup_context {
     echo >&2 "Kubernetes cluster URL is missing"
     exit 1
   fi
-
+  
   if [[ -z $Octopus_AccountType && -z $Octopus_K8S_Client_Cert && ${Octopus_EKS_Use_Instance_Role,,} != "true" ]]; then
-    echo >&2 "Kubernetes account type or certificate is missing"
-    exit 1
+    if [[ -z $Octopus_K8s_Pod_Service_Account_Token_Path && -z $Octopus_K8s_Server_Cert_Path ]]; then
+      echo >&2 "Kubernetes account type or certificate is missing"
+      exit 1
+    fi
+    
+    Octopus_K8s_Pod_Service_Account_Token=$(cat ${Octopus_K8s_Pod_Service_Account_Token_Path})
+    Octopus_K8s_Server_Cert_From_Path=$(cat ${Octopus_K8s_Server_Cert_Path})
```

Comment:
The cert path technically doesn't have to be defined. It may be possible for the K8s API to have a trusted CA certificate for example. It would be worth checking if `Octopus_K8s_Server_Cert_Path` is empty before trying to cat it. I'm not sure what `cat` will do with an empty argument.

Reply:
> I'm not sure what cat will do with an empty argument.
it returns empty :) but I'll just add a check just to be safe.